src/haxeLanguageServer/Context.hx

```diff
@@ -252,6 +256,13 @@ class Context {
 			capabilities.foldingRangeProvider = true;
 		}
 
+		if (textDocument!.colorProvider!.dynamicRegistration == true) {
+			register(DocumentColorRequest.type, haxeSelector);
+			register(ColorPresentationRequest.type, haxeSelector);
```

Comment:
registering `ColorPresentationRequest` gives a `Request client/registerCapability failed with message: No feature implementation for textDocument/colorPresentation found. Registration failed.` during startup of languageserver. 

to quote the [spec](https://microsoft.github.io/language-server-protocol/specification#textDocument_colorPresentation):
> This request has no special capabilities and registration options since it is send as a resolve request for the [textDocument/documentColor](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_documentColor) request.